js/LPJsonPollock.js

```diff
@@ -94,18 +94,11 @@ export default class LPJsonPollock {
     }
   }
 
-  render(json: Object|string): DocumentFragment {
+  render(json: Object|string, options: Object = {}): DocumentFragment {
     let jsonObj: Object;
-    if (Utils.isString(json)) {
-      // This will throws an error if fails
-      jsonObj = JSON.parse((json: any));
-    } else {
-      jsonObj = (json: any);
-    }
-    this.jsonValidator(jsonObj);
-    if (this.jsonValidator.errors) {
-      throw new JsonPollockError('Schema validation error, see \'errors\' for more details', this.jsonValidator.errors);
-    }
+	  if (!options.skipValidation) {
```

Comment:
Now the validation function is also parse the json object if it was given as a string, so if the `skipValidation` option is enabled we missed that part

Reply:
true! i fix!

Reply:
done!